api/v1/storagecluster_types.go

```diff
@@ -105,6 +105,11 @@ type StorageClusterSpec struct {
 	DefaultStorageProfile string `json:"defaultStorageProfile,omitempty"`
 
 	StorageProfiles []StorageProfile `json:"storageProfiles,omitempty"`
+
+	// +kubebuilder:validation:MaxLength=230
+	// StorageClassName is the string to be used as the prefix for
+	// the default storage classes created by ocs-operator.
+	StorageClassName string `json:"storageClassName,omitempty"`
```

Comment:
If someone changes the prefix after creating the storage cluster, what will be the impact? Should we prevent such operations from happening in the first place?

Reply:
In that case, the operator will reconcile and create new storage classes with the updated names.
IMO, we should allow them to change the name if they want to, in that way they can delete older sc's if they are unused and create new one's.